content/_changelogs/9.5.2.md

```diff
@@ -0,0 +1,42 @@
+## 9.5.2
+
+_Released 3/14/2022_
+
+**Bugfixes:**
+
+- Fixed an issue with `.type()` where click events could be fired on the
+  incorrect target element because the target focus changed within a key-down
+  event handler callback. Fixed in
+  [#20525](https://github.com/cypress-io/cypress/pulls/20525).
+- Fixed a regression in [9.5.0](/guides/references/changelog#9-5-0) where ANSI
+  colors were not removed from the FireFox warning message about the
+  `chromeWebSecurity` configuration option having no effect on the Firefox
+  browser. Fixes [#20496](https://github.com/cypress-io/cypress/issues/20496).
+- Updates were made to the pre-release build setup such that Cypress will use a
+  unique cache folder for each
+  [pre-release installation](../guides/getting-started/installing-cypress.md#Install-pre-release-version)
```

Comment:
Links to internal docs are not relative. They should be a path that corresponds with a corresponding guide as defined in our doc structure. 
```suggestion
  [pre-release installation](/guides/getting-started/installing-cypress.md#Install-pre-release-version)
```

Reply:
oof. I forgot... we should writing a markdown linter rule to check this